client/.vscodeignore

```diff
@@ -6,4 +6,7 @@ src/**
 **/*.map
 .gitignore
 tsconfig.json
-vsc-extension-quickstart.md
+tmp/**
+logs/**
+sessions/**
+gulpfile.js
```

Comment:
logs and sessions was from the PS module.  tmp and gulpfile I found while packaging.